pkg/controller/servicebindingrequest/bind_custom_resources_test.go

```diff
@@ -72,3 +73,64 @@ func TestBindNonBindableResources_GetOwnedResources(t *testing.T) {
 		require.Equal(t, 3, len(data), "")
 	})
 }
+
+func TestBindNonBindableResources_Secret_GetOwnedResources(t *testing.T) {
+	f := mocks.NewFake(t, "test")
+	cr := mocks.DatabaseCRMock("test", "test")
+	trueBool := true
+	reference := v1.OwnerReference{
+		APIVersion:         cr.APIVersion,
+		Kind:               cr.Kind,
+		Name:               cr.Name,
+		UID:                cr.UID,
+		Controller:         &trueBool,
```

Comment:
Can you create a `TruePtr` and `FalsePtr` constants (or equivalent) and make them available to be used globally?

Reply:
+1